src/views/about.html

```diff
@@ -3,6 +3,7 @@
 
 <head>
     <meta name="description" content="This is the about page for Aggie Coding Club.">
+    <title>Aggie Coding Club - About</title>
     <meta charset="UTF-8">
     <title>Aggie Coding Club - About</title>
```

Comment:
```suggestion
    <meta charset="UTF-8">
    <title>Aggie Coding Club - About</title>
```
We only need it one time